mitiq/shadows/shadows.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (C) Unitary Fund
+#
+# This source code is licensed under the GPL license (v3) found in the
+# LICENSE file in the root directory of this source tree.
+"""Classical shadow estimation for quantum circuits. Based on the paper"""
```

Comment:
Add reference to paper, or remove second sentence.

Reply:
thanks, have deleted it.